### ISSUE TITLE
Release new patch version to fix plugin org.dita.pdf2.fop security issues

### ISSUE THREAD

**Issue by ghost** [dita-ot/dita-ot#4321]
Plugin `org.dita.pdf2.fop` included in the latest 4.1.2 release includes `batik-all-1.16.jar` dependency, which has 2 security issues:
* [CVE-2022-44730](https://nvd.nist.gov/vuln/detail/CVE-2022-44730) (4.4 medium severity)
* [CVE-2022-44729](https://nvd.nist.gov/vuln/detail/CVE-2022-44729) (7.1 high severity)

Both of these can be solved by updating `batik` dependency to version 1.17, which is already fixed in `develop` branch.

Could a patch version be released with this change included to fix the security issues?

**Comment by jelovirt** [comment]
To release this in 4.1.3 we'd need to verify Batik 1.17 is compatible with FOP 2.8.

**Comment by raducoravu** [comment]
Maybe copy the changes from #4272 to the 4.1.3 hot fix branch?

**Comment by Roopesh-Navath** [comment]
I am using Antennahouse to generate PDF, Is this safe to remove the plugin **org.dita.pdf2.fop** from DITA OT?

**Comment by jelovirt** [comment]
@Roopesh-Navath Yes, you don't need org.dita.pdf2.fop if you're using org.dita.pdf2.axf or Antennahouse's PDF5

**Comment by super-structure** [comment]
> To release this in 4.1.3 we'd need to verify Batik 1.17 is compatible with FOP 2.8.

It does appear to be compatible with FOP 2.8. Tested with swapping it, updating references to the .jar, and publishing a PDF with an SVG figure. Not comprehensive testing, but promising.
